pygmt/src/velo.py

```diff
@@ -0,0 +1,252 @@
+"""
+velo - Plot velocity vectors, crosses, anisotropy bars and wedges.
+"""
+import numpy as np
+import pandas as pd
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    A="vector",
+    B="frame",
+    C="cmap",
+    D="rescale",
+    E="uncertainty_color",
+    G="facecolor",
+    H="scale",
+    I="intensity",
+    J="projection",
+    L="line",
+    N="no_clip",
+    R="region",
+    S="scaling",
+    U="timestamp",
+    V="verbose",
+    W="pen",
+    X="xshift",
+    Y="yshift",
+    Z="zvalue",
+    c="panel",
+    p="perspective",
+    t="transparency",
+)
+@kwargs_to_strings(R="sequence", c="sequence_comma", i="sequence_comma", p="sequence")
+def velo(self, data=None, **kwargs):
+    r"""
+    Plot velocity vectors, crosses, anisotropy bars and wedges.
+
+    Reads data values from files, numpy array or pandas DataFrame and will plot
+    the selected geodesy symbol on a map. You may choose from velocity vectors
+    and their uncertainties, rotational wedges and their uncertainties,
+    anisotropy bars, or strain crosses. Symbol fills or their outlines may be
+    colored based on constant parameters or via color lookup tables.
+
+    Must provide *data* and *scaling*.
+
+    Full option list at :gmt-docs:`supplements/geodesy/velo.html`
```

Comment:
I'm thinking whether to mention the ["Data Column Order"](https://docs.generic-mapping-tools.org/dev/supplements/geodesy/velo.html#data-column-order) documentation here since the order of the fields matter:

```
lon lat symbol-columns [usercol] [size] [scale] [intens] [transp [transp2]] [trailing-text]
```

Reply:
I would prefer that an issue is opened to track a `velo` tutorial that covers the material in "Data Column Order", since it would be better to document the material using PyGMT's aliases.

Related to this is that "Data Column Order" documentation section is new in GMT 6.2 and includes new options from 6.2 that do not have aliases in this PR. These could be added in a separate PR after 6.2 release.

Reply:
Ok, will leave that to be in a separate tutorial then.